zarr.qmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Zarr"
+subtitle: Chunked, Compressed N-Dimensional Arrays
+---
+
+# What is Zarr?
+
+[Zarr](https://zarr.dev/), despite its name, is not a scary format. It's designed for data that is too big for users' machines, but Zarr makes data small and organizes it in a way where users can take just the bits they need or distribute the load of processing lots of those bits (stored as chunks) across many machines.
+
+The Zarr data format is a community-maintained format for large-scale n-dimensional data. A Zarr store consists of compressed and chunked n-dimensional arrays. Zarr's flexible indexing and compatibility with object storage lends itself to parallel processing.
+
+Read more in the official tutorial: [Zarr Tutorial](https://zarr.readthedocs.io/en/stable/tutorial.html)
+
+## Zarr Version 2 and Version 3
```

Comment:
Especially since Version 3 hasn't been released yet, I'd tend to suggest moving the version differences section after the general Zarr overview